verilog/CST/verilog_matchers.h

```diff
@@ -298,6 +298,12 @@ static const auto AlwaysCombKeyword =
 static const auto AlwaysFFKeyword =
     verible::matcher::MakePathMatcher({L(TK_always_ff)});
 
+// Matches occurrence of the 'always' keyword.
```

Comment:
matches occurence of 'always' keyword ? Sounds like a copy/paste situation here :)